src/reducers/versions.tsx

```diff
@@ -0,0 +1,218 @@
+import { Reducer } from 'redux';
+import { ActionType, createAction, getType } from 'typesafe-actions';
+
+type VersionId = number;
+
+type LocalizedString = {
+  [lang: string]: string;
+};
```

Comment:
I have the feeling that this could be reused somewhere else in the future, but no need to change anything right now

Reply:
I was thinking the same thing, about both. :-)